datahub-web-react/src/app/entityV2/summary/documentation/AboutContent.tsx

```diff
@@ -46,6 +47,8 @@ export default function AboutContent() {
         handleCancel,
         emptyDescriptionText,
     } = useDescriptionUtils();
+    const { entityData } = useEntityData();
+    const canEditDescription = !!entityData?.privileges?.canEditDescription;
```

Comment:
unfortunately for glossary terms and nodes we still do the check `GlossaryUtils.canManageChildrenEntities` inside of `UpdateDescriptionResolver` so i think we will have to do something similar to the links where we also check that permission